edb/lib/cfg.edgeql

```diff
@@ -178,6 +178,13 @@ ALTER TYPE cfg::AbstractConfig {
             Access-Control-Allow-Origin HTTP header';
     };
 
+    CREATE PROPERTY enable_recompilation -> std::bool {
+        SET default := true;
+        CREATE ANNOTATION cfg::system := 'true';
+        CREATE ANNOTATION std::description :=
+            'Recompile all cached queries on DDL if enabled.';
+    };
```

Comment:
This shouldn't need to be system, right? It could be compiled at any level and should be fine, and then we could disable/enable on specific test suites more easily